src/playlist-loader.js

```diff
@@ -395,7 +395,7 @@ export default class PlaylistLoader extends EventTarget {
 
     this.customTagParsers = (vhsOptions && vhsOptions.customTagParsers) || [];
     this.customTagMappers = (vhsOptions && vhsOptions.customTagMappers) || [];
-    this.llhls = (vhsOptions && vhsOptions.llhls) || true;
+    this.llhls = (vhsOptions && vhsOptions.llhls) || false;
```

Comment:
Since the option is set at a higher level, I think we can just use:

```suggestion
    this.llhls = vhsOptions && vhsOptions.llhls;
```